nise/generators/azure/azure_generator.py

```diff
@@ -25,43 +25,88 @@
 
 from nise.generators.generator import AbstractGenerator
 
-AZURE_COLUMNS = (
-    "SubscriptionGuid",
-    "ResourceGroup",
+AZURE_COLUMNS_V2_SUBSCRIPTION = (
```

Comment:
Same issue here: this is changing something in the public interface which will break anyone using this global. That may be fine, but it's something to consider.

Reply:
This is a good point and I suggest we discuss it.